python/tk_desktop/thumb_widget.py

```diff
@@ -34,11 +36,25 @@ def __init__(self, width=120, parent=None):
 
     def set_thumbnail(self, pixmap):
         """Set a thumbnail given the current pixmap."""
-        # zoom to fit height, then crop to center
-        pixmap = pixmap.scaledToHeight(self.thumb_size, QtCore.Qt.SmoothTransformation)
-        if pixmap.width() > self.thumb_size:
-            extra = pixmap.width() - self.thumb_size
-            pixmap = pixmap.copy(extra / 2, 0, self.thumb_size, self.thumb_size)
+
+        if os.getenv("SGTK_PROJ_THUMB_OLD") == "True":
+            # zoom to fit height, then crop to center
+            pixmap = pixmap.scaledToHeight(
+                self.thumb_size, QtCore.Qt.SmoothTransformation
+            )
+            if pixmap.width() > self.thumb_size:
+                extra = pixmap.width() - self.thumb_size
+                pixmap = pixmap.copy(extra / 2, 0, self.thumb_size, self.thumb_size)
+        else:
+            dst_pixmap = QtGui.QPixmap(self.thumb_size, self.thumb_size)
```

Comment:
Is it possible to add a comment to describe what is the strategy (scale?) in this else block? 
Like the `zoom to fit height, then crop to center` in the IF block

Reply:
@julien-lang , I believe this was addressed, can we resolve this conversation? Thanks!